exec.go

```diff
@@ -124,20 +124,17 @@ func getSubCgroupPaths(args []string) (map[string]string, error) {
 	paths := make(map[string]string, len(args))
 	for _, c := range args {
 		// Split into controller:path.
-		cs := strings.SplitN(c, ":", 3)
-		if len(cs) > 2 {
-			return nil, fmt.Errorf("invalid --cgroup argument: %s", c)
-		}
-		if len(cs) == 1 { // no controller: prefix
+		if ctr, path, ok := strings.Cut(c, ":"); ok {
```

Comment:
I don't check this will lead to some regressions, for example, if c == "a:b:c:d:e":
When using `SplitN`, `cs[1]` will be "b";
When using `Cut`, `path` will be "b:c:d:e".

Reply:
So, the old code would error out when `c == "a:b:c:d:e"`.

The new code would not, assigning `b:c:d:e` to `path`.

To me, the new code is (ever so slightly) more correct, since `:` is a valid symbol which should be allowed in `path`.

Reply:
Yup, was looking at that as well; agreed, looks like new code is more correct here.


I still need to have a quick look at the splitting on commas, and the `if len(args) != 1 {` check below. Probably will check out this branch locally